worlds/tloz/Rom.py

```diff
@@ -71,9 +71,5 @@ def get_base_rom_bytes(file_name: str = "") -> bytes:
 
 
 def get_base_rom_path(file_name: str = "") -> str:
```

Comment:
argument is unused now.

I think the 2 solutions would be either getting rid of the argument, or restoring the use of the argument in the code, i.e. `if file_name and os.path.exists(file_name): return file_name`

I'm gonna say if there is no reason for the argument to exist, it's probably better to remove it.